prov/efa/src/efa_cq.h

```diff
@@ -3,6 +3,82 @@
 
 #include "efa.h"
 
+enum ibv_cq_ex_type {
+	IBV_CQ,
+	EFADV_CQ
+};
+
+struct efa_ibv_cq {
+	struct ibv_cq_ex *ibv_cq_ex;
+	enum ibv_cq_ex_type ibv_cq_ex_type;
+	/* dlist entry inserted to cq/cntr's poll_list */
+	struct dlist_entry entry;
```

Comment:
Why do we need to make efa_ibv_cq a linked list?

Reply:
Good catch, the `entry` should be removed. It was added  when I experimented another approach to implement that ibv_cq_poll_list but I failed to clean up it.